Dockerfile.builder

```diff
@@ -0,0 +1,12 @@
+ARG DOCKER_ARCH=amd64
+
+# Used to build the wal2json library in a GHA. This image should 
+# almost never change. We do not need to build it every time.
+# We would mount the working directory with the source (that's the
+# part that changes) and run the build process on a running container
+
+FROM $DOCKER_ARCH/postgres:9.6-alpine
+RUN apk add --no-cache --virtual .build-deps gcc git make musl-dev pkgconf diffutils
```

Comment:
Out of curiosity, why do we need `git` to compile stuff?

Reply:
I don't think it does. I picked this list from the other Dockerfile we have that builds the library. Will try to remove it and see if it is ok.

Reply:
The lighter the Docker image, the better.